client/src/components/requests/RequestedList.tsx

```diff
@@ -0,0 +1,151 @@
+import React from "react";
+import {
+    Card,
+    Container,
+    Header,
+    Label,
+    Step
+} from "semantic-ui-react";
+import {RequestedItem} from "../../types/Hardware";
+import {useQuery} from "@apollo/react-hooks";
+import {DESK_REQUESTS} from "../util/graphql/Queries";
+import {Request} from "../../types/Request";
+
+interface RequestedListProps {
+    requestedItemsList: RequestedItem[],
+}
+
+function RequestedList({requestedItemsList}: RequestedListProps) {
+    const {loading, error, data} = useQuery(DESK_REQUESTS);
+    if (loading) return <p>Loading...</p>;
+    if (error) return <p>Error :(</p>;
```

Comment:
Render `error.message` in a Semantic `Message` component with `danger`/red styling pls.  (You can base this off the hardware inventory list error message)